tmt/steps/provision/local.py

```diff
@@ -51,13 +51,14 @@ def requires(self):
 class GuestLocal(tmt.Guest):
     """ Local Host """
 
-    def ansible(self, playbook):
+    def ansible(self, playbook, extra_args):
```

Comment:
I think `extra_args` should not be a mandatory parameter, let's make it default to `None`. Similar in other places.